_j/constraints.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Removing a package based on a constraint
+---
+
+One or multiple packages were not considered during the resolution based on
+constraints supplied.
+
+## Issue description
+
+Constraints provide a mechanism to state dependencies that are known to be
+working without a need to adjust direct dependency requirements. This allows to
```

Comment:
"This permits restricting requirements..."

Reply:
Thanks for the review! 👍🏻 Adjusted in c0b9dcde